plothist/hep_plotters.py

```diff
@@ -67,8 +61,8 @@ def compare_data_mc(
         The main axes for the histogram comparison. If fig, ax_main and ax_comparison are None, a new axes will be created. Default is None.
     ax_comparison : matplotlib.axes.Axes or None, optional
         The axes for the comparison plot. If fig, ax_main and ax_comparison are None, a new axes will be created. Default is None.
-    ratio_uncertainty : str, optional
-        How to treat the uncertainties of the histograms when comparison = "ratio" ("uncorrelated" for simple comparison, "split" for scaling and split hist_1 and hist_2 uncertainties). Default is "split".
+    **comparison_kwargs : optional
+        Arguments to be passed to plot_comparison(). By default for compare_data_mc(), x1_label="Data", x2_label="Pred.", comparison="ratio", and ratio_uncertainty="split".
```

Comment:
By default for compare_data_mc(), --> If they are not provided explicitly, the following arguments are passed by default: